README.md

```diff
@@ -26,6 +26,8 @@ Parse a HTTP `Cookie` header string and returning an object of all cookie name-v
 The `str` argument is the string representing a `Cookie` header value and `options` is an
 optional object containing additional parsing options.
 
+**NOTE:** This method does not parse the `set-cookie` header. It parses the `cookie` header. This means you cannot do `cookie.parse(cookie.serialize('name', 'value'))`.
```

Comment:
```suggestion
**Note:** This method does not parse the `Set-Cookie` header.
```

It already says it parses the `Cookie` header twice in the above paragraph, would prefer to keep the note shorter so someone skimming actually reads it.

Reply:
lmao I went in the opposite direction with my suggestion.

Reply:
I'd split the difference

`> ⚠️ **Note:** This method does not parse the `Set-Cookie` header.`

Reply:
I am personally in favor of a more detailed description. Honestly I would prefer to talk about the difference and even show a detailed example (on top of what your suggestion is below). The larger we make the block warning for this the more likely someone is to see it IMO. Replying here since this is the one with the discussion, but I lean toward @jonchurch's direction below.